### PR TITLE
HackUPC 2018 edition style

### PR DESCRIPTION
**Pull request by oriolclosa**
## What this PR does / Why we need it
This PR changes the default style according to the next edition of HackUPC, it also introduces some minor changes such as added fields on registration or a new checkbox for Terms and Conditions.

1. Changed image on Readme and fixed e-mail confirmation template reference.
2. Main stylesheet changed for the new edition, changed site title on header and some information in it.
3. Code of Conduct changed according to HackUPC own Code of Conduct.
4. Terms and Condition page added with blank text (will do another PR or modify this one when I have the text available).
5. Universities list json updated with all Catalan Countries universities added.
6. Added a field on registration (non required) for us to collect some information in what hardware hackers want.
7. Added a checkbox (same as the one for Code of Conduct) for the Terms and Conditions linking to the new page (by now empty).
8. Changed e-mail templates according to the new edition style.
9. Social networks changed on the footer of the page as well as on the e-mails templates (and so new variables have been added).
10. And most importantly, added a laptop on the footer along with the heart, plase keep it, I'm also a sentimental :relaxed: (and it's on the new frontend version as well).

## Special notes for your reviewer (optional)
Pls, accept this pull request :smile:.

## Some questions
- [X] I have read the contributing guidelines
- [X] I abide by this repository Code of Conduct
- [X] I understand that my PR won't be merged until Travis gives a "green light"

## Additional Notes (optional)
Changes introduced in the last version were astonishing, really impressive!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
-<br>
 <p align="center">
-  <img alt="HackUPC Fall 2016" src="https://github.com/hackupc/frontend/raw/master/src/images/hackupc-header.png" width="620"/>
+  <img alt="HackUPC Fall 2016" src="app/static/img/hackupc-ogimage@2x.png" width="100%"/>
```
Comment (ofpau) [inline README.md]: The alt text is outdated

File: app/hackathon_variables.py [hunk 2]
```diff
@@ -9,33 +9,39 @@
 # Hackathon timezone
 TIME_ZONE = 'CET'
 # This description will be used on the html and sharing meta tags
-HACKATHON_DESCRIPTION = 'Join us for BarcelonaTech\'s hackathon. 700 hackers. 36h. October 13th-15th.'
+HACKATHON_DESCRIPTION = 'Join us for BarcelonaTech\'s hackathon. 700 hackers. 36h. October 5th-7th.'
```
Comment (ofpau) [inline app/hackathon_variables.py]: :cry:

File: app/templates/code_conduct.html [hunk 3]
```diff
@@ -1,76 +1,42 @@
...
-{% block head_title %}Code of conduct{% endblock %}
+{% block head_title %}Code of Conduct{% endblock %}
 
 {% block head %}
     <link rel="stylesheet" href="{% static 'css/main.css' %}">
 {% endblock %}
 
 {% block panel %}
+    <h1>{{ h_name }} Code of Conduct</h1>
+    <p>This is the Code of Conduct that we expect all HackUPC participants to abide. This is a customized version of the Code of Conduct used at MLH Member Hackathons. If you have a suggestion for this code of conduct or a question about them, please submit an issue <a href="https://github.com/hackupc/rules/issues" target="_blank">here</a>.</p>
```
Comment (ofpau) [inline app/templates/code_conduct.html]: _to abide **by**_

File: applications/static/schools2.json [hunk 4]
```diff
@@ -7519,5 +7519,26 @@
...
+"Universitat de Perpinyà Via Domícia - UPVD",
+"Universitat Politècnica de València - UPV",
+"Universitat Ramon Llull - URL",
+"Universitat Rovira i Virgili - URV",
+"Universitat de València - UV",
+"Universitat de Vic - UVic",
+"Universitat Catòlica de València San Vicente Mártir - UCV",
+"Universidad Cardenal Herrera CEU",
+"Valencian International University - VIC"
 ]
```
Comment (ofpau) [inline applications/static/schools2.json]: Nice :+1:

File: app/templates/code_conduct.html (hunk 3 above)
Comment (casassg) [reply app/templates/code_conduct.html]: This should be changed on the code of conduct first

**Comment by oriolclosa** [comment]
:heart:

**Pull request merged**